src/core/tests/test_font.py

```diff
@@ -38,3 +44,9 @@ def test_variant(self):
 
     def test_weight(self):
         self.assertEqual(self.font.weight, self.weight)
+
+    def test_register(self):
```

Comment:
There's a few other use cases here that should be tested - especially on the fallback lookup path.

Reply:
OK, there is no fallback lookup path anymore